code/modules/spells/spell_types/lichdom.dm

```diff
@@ -154,3 +158,25 @@
 
 
 	return "Respawn of [mind] successful."
+
+/obj/effect/proc_holder/spell/self/tap
```

Comment:
Duplicate definitions?

Reply:
dunno how this got left in, let me handle that